packer/provisioner.go

```diff
@@ -40,6 +42,10 @@ type ProvisionHook struct {
 	// The provisioners to run as part of the hook. These should already
 	// be prepared (by calling Prepare) at some earlier stage.
 	Provisioners []*HookedProvisioner
+
+	ParentBuildName string
+	ParentBuildType string
```

Comment:
I'm unsure if the notion of Parent is correct here, could these simply be `BuildName` and `BuildType` ?